.yamato/standalone-project-tests.yml

```diff
@@ -0,0 +1,35 @@
+{% metadata_file .yamato/project.metafile %}
+---
+
+# For every platform and editor version, run its project tests without
+# running package tests too since they are handled on their respective
+# jobs
+{% for project in projects -%}
+{% for editor in project.test_editors -%}
+{% for platform in test_platforms -%}
+standalone_test_{{ project.name }}_{{ editor }}_{{ platform.name }}:
+  name : standalone {{ project.name }} tests - {{ editor }} on {{ platform.name }}
+  agent:
+    type: {{ platform.type }}{% if platform.name == "ubuntu" %}::GPU{% endif %}
```

Comment:
Why are you asking for a GPU instance with an rtx2080 if the actual test command uses `-batchmode -nographics`?

Reply:
The build doesn't need it, but the run time crashes without a real graphics context.

Reply:
Should I separate build and test into different jobs?

Reply:
Ah, @JesseOlmer I finally have a better answer. The issue is that I'm launching the standalone player and the extra-editor-args don't get passed to that so the player still comes up and attempts to launch the ui which it can't. I'm asking around as to how to get the player to take args through utr but I don't know what else to do at the moment.